docs/guides/developer/docs/api/events-api.md

```diff
@@ -919,3 +920,14 @@ In case of a conflict:
   }
 ]
 ```
+
+`allowConflict` was introduced in version 1.2.0, it allows the schedule to be
```

Comment:
I would omit "was introduced in version 1.2.0, it" as the table above already indicates that.

Reply:
Ok, will remove.